app/src/main/java/com/github/polybooks/AddSaleActivity.kt

```diff
@@ -7,13 +7,14 @@ import android.text.TextWatcher
 import android.view.View
 import android.widget.EditText
 import androidx.appcompat.app.AppCompatActivity
-import com.github.polybooks.utils.setupNavbar
 import com.github.polybooks.utils.StringsManip.isbnHasCorrectFormat
 import com.github.polybooks.utils.UIManip.disableButton
 import com.github.polybooks.utils.UIManip.enableButton
+import com.github.polybooks.utils.setupNavbar
 
 
-const val ISBN = "com.github.polybooks.ISBN"
+const val EXTRA_ISBN = "com.github.polybooks.ISBN"
+const val EXTRA_PICTURE_FILE = "com.github.polybooks.picture_file"
```

Comment:
in my sense, those should be in the class as companion object, but I let you decide

Reply:
They're definitely misplaced, you're right. 
But as I was working on the global variable object, I thought of moving them there in the future, as they're not related to this activity but rather to extras between 3-4 activities.